src/orange/orangeinp/ConvexRegion.cc

```diff
@@ -633,6 +633,15 @@ void Sphere::output(JsonPimpl* j) const
     to_json_pimpl(j, *this);
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Whether this encloses another sphere.
+ */
+bool Sphere::encloses(Sphere const& other) const
```

Comment:
Why this enclosing test does not account for the azimuthal wedge?
E.g. is a larger-radius sphere with azimuthal edge supposed to enclose a concentric, smaller-radius sphere without an azimuthal edge?

Reply:
This is just for the "convex region", the sphere itself. It's only use for verification when building a solid. It's not used for checking solid enclosures.